### PR TITLE
remove crio bridge file

### PR DESCRIPTION
**Pull request by cooktheryan**
Removal of CRI-O bridge file as we have not been documenting using it

Signed-off-by: Ryan Cook <rcook@redhat.com>

### CODE REVIEW

**Comment by cooktheryan** [comment]
@oglok I'm going to remove this

File: ci-operator/config/redhat-et/microshift/redhat-et-microshift-main.yaml [hunk 1]
```diff
@@ -42,8 +41,6 @@ images:
         source_path: /go/src/github.com/redhat-et/microshift/packaging/systemd/microshift-containerized
       - destination_dir: .
         source_path: /go/src/github.com/redhat-et/microshift/packaging/systemd/microshift-containerized.service
-      - destination_dir: .
-        source_path: /go/src/github.com/redhat-et/microshift/packaging/images/microshift-aio/crio-bridge.conf
```
Comment (mangelajo) [inline ci-operator/config/redhat-et/microshift/redhat-et-microshift-main.yaml:46]: we should instead copy packaging/crio.conf.d/microshift.conf to /etc/crio/crio.conf.d

Comment (oglok) [reply ci-operator/config/redhat-et/microshift/redhat-et-microshift-main.yaml:46]: agreed, that file was merged yesterday to wait for network creation.

**Review by mangelajo — COMMENTED** [review]
/lgtm

**Pull request merged**